### PR TITLE
add metrics files for rnafusion

### PR DESCRIPTION
**Pull request by fevac**
## Description
- Adds metrics-qc and multiqc-json tags for rnafusion

### How to prepare for test
- [ ] ssh to hasta.scilifelab.se
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_hermes -t hermes -b add_metrics_to_deliverables`

### How to test
- [ ] Do ...

### Expected test outcome
- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch
- [ ] Inform to ...

### CODE REVIEW

**Review by ivadym — APPROVED** [review]
Nice 💯

File: cg_hermes/config/rnafusion.py
[the remark this answers is not in the document]
Comment (diitaz93) [reply cg_hermes/config/rnafusion.py:1]: Is it `RNAFUSION_COMMON_TAGS: Dict[set, Dict[str, Any]]` something like that?

**Comment by fevac** [comment]
- deployed in stage:
image


- deployed in prod:

image

File: cg_hermes/config/rnafusion.py
[the remark this answers is not in the document]
Comment (fevac) [reply cg_hermes/config/rnafusion.py:1]: shoot! I just realized I merged before pushing my local changes. So no type hints have been added to master. I won't open another PR for this but I'll try to remember next time I'm changing these tags

**Pull request merged**